### ISSUE TITLE
Problem to print posts or topic

### ISSUE THREAD

**Issue by alfazaz** [NodeBB/NodeBB#8033]
Hello !

Like I said on [this topic on nodebb community forum](https://community.nodebb.org/topic/14338/how-to-print-correctly-some-posts-a-topic?_=1573389617178), printing a post, many posts selected or all posts in a topic is really not easy or it doesn't work (tried it on firefox 70 for example).

I think css should be adapted to have a good printing. There could also exist a post/topic tool to be able to directly print a post, selected posts and all topic posts (and it should work even with infinite scroll) ; this last point can be an improvement of nodebb (and not only nodebb persona).

Best Regards,
Alfazaz

**Comment by julianlam** [comment]
While technically a theme issue, I feel this can be better served in the main issue tracker for NodeBB.

**Comment by alfazaz** [comment]
Yes, I think too... Must I open an issue also on nodebb github ?

**Comment by alfazaz** [comment]
Yer, I think too... I see that you moved this to nodebb/nodebb github. Thanks !

**Comment by mySebbe** [comment]
I rechecked current core plus the current Harmony and Persona branches. There is still no print route/tool and neither official theme has print-specific CSS. A CSS-only first change can make the currently rendered posts printable, but it cannot fetch an entire infinite-scroll topic or define a selected-post document. Would you want the first scope to be only “print the posts currently rendered by the active theme,” with “whole topic” and “selected posts” tracked separately? That decision determines whether this belongs in the themes or needs a new server-rendered print endpoint.
